### PR TITLE
Update to run script 2.4.1

### PR DESCRIPTION
**Pull request by nottrobin**
QA
--

```
./run
./run build
./run clean
.. etc.
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,14 +15,15 @@
...
-    "node-sass": "^4.5.3",
-    "postcss-cli": "^4.1.0",
+  "dependencies": {
+    "watch-cli": "^0.2.2",
     "sass-lint": "^1.10.2",
+    "postcss-cli": "^4.1.0",
+    "node-sass": "^4.5.3",
+    "cssnano": "^3.10.0",
+    "autoprefixer": "^6.3.1",
+    "autoprefixer": "^6.3.1",
```
Comment (WillMoggridge) [inline package.json]: The change to this file adds autoprefixer twice, unorders the alphabetical items, and changed devDependencies to dependencies. :(

Comment (nottrobin) [reply package.json]: @WillMoggridge what's wrong with changing devdependencies to dependencies?

Comment (nottrobin) [reply package.json]: @WillMoggridge https://github.com/canonical-webteam/generator-canonical-webteam/blob/master/generators/run/templates/package.json#L8

Comment (nottrobin) [reply package.json]: Fixed the rest. Left "dependencies".

**Pull request merged**